### ISSUE TITLE
[Request] - Liquid XP Generator

### ISSUE THREAD

**Issue by JaSpr** [rwtema/extrautilities_old_issues#1488]
I've run into multiple situations on various servers running various modpacks where we obtained an absurd amount of XP that could never be used.  Probably couldn't even use 5% of it.  It would be nice to have an XP generator powered off of Liquid XP.

**Comment by MyBigFan2** [comment]
True, I created my own mod pack and I have to use a exp solidifier from actullay additions.
Ended up filling my storage system.
